index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="main-container">
+            <div class="header-container">
+              <nav class="header">
+                <div class="my-logo">
+                  <a class="Logo" href="#">My Logo</a>
+                </div>
+                <div class="hambuger">
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                </div>
+              </nav>
+            </div>
+        </div>
+    </header>
+    <section class="img-BG">
+        <div class="main-container">
+            <div class="portfolio-description">
+                <div class="name">
+                    <h1>I'm Ifeanyi, Glad to see you!</h1>
+                </div>
+                <div class="my-qualities">
+                  <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="lets-connect">
+                  <p>LET’S CONNECT</p>
+                </div>
+                <ul class="social-media">
+                    <li><a href="#"><img src="images/twitter-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/Linkedin icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/M-logo.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/github-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/index-finger.svg" alt=""></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
```

Comment:
- This section in your website design doesn't match the one in the Figma template. Please ensure that 'Let's connect" aligns with the rest of the text above,  note that you need to try to replicate the design as closely s possible. This will give confidence to recruiters in your ability to replicate exact design templates. 🙏 


Your design :
![lets connect project](https://user-images.githubusercontent.com/73607512/191863666-1848b0c9-fc2d-41ed-ab20-d73ea2ae67cc.PNG)


Figma template:
![lets connect figma](https://user-images.githubusercontent.com/73607512/191863736-3dbe4866-877a-4211-a07f-6ff92cab4b20.PNG)

Reply:
Hi, @mckent05 Thanks for the heads up,
I'll add the required changes to my project